int32 obs_idx
float64 obs_size
int32 goal_idx
int32 big_goal_radius
int32 sparse
---
float64[] states